<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphologyOperationsWidget</class>
 <widget class="QWidget" name="MorphologyOperationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MorphologyOperationsCanvas" name="morphologyOperationsCanvas" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>571</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dylationButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Dylatacja</string>
   </property>
  </widget>
  <widget class="QPushButton" name="erosionButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Erozja</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyczyść</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MorphologyOperationsCanvas</class>
   <extends>QWidget</extends>
   <header>morphologyoperationscanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
